src/attributes/classes.js

```diff
@@ -78,7 +76,7 @@ jQuery.fn.extend({
 
 				if ( cur ) {
 					j = 0;
-					while ( (clazz = classes[j++]) ) {
+					while ( ( clazz = classes[ j++ ] ) ) {
```

Comment:
I search about this and it's in all the jquery.js every where, but it doesnt appear in the minified version.